package.json

```diff
@@ -55,119 +55,69 @@
               "altText": "Project Explorer"
             },
             "when": "java:projectManagerActivated"
           },
           {
-            "id": "java.showProjectExplorer.notActivated",
+            "id": "java.showProjectExplorer.inactive",
             "title": "Explore your project",
-            "description": "View classpaths and dependencies, and create new projects, packages and classes.",
+            "description": "%walkthrough.java.projects.inactive%",
             "media": {
               "image": "walkthrough/project-manager.png",
               "altText": "Project Explorer"
             },
             "when": "!java:projectManagerActivated"
           },
           {
-            "id": "java.showDebugView",
-            "title": "Launch and debug your app",
-            "description": "The debugger will automatically find the entry point of your project, launch it and start debugging.\n[Show Run and Debug View](command:workbench.view.debug)\n[Learn More...](https://code.visualstudio.com/docs/java/java-debugging)",
+            "id": "java.debugAndTest",
+            "title": "Launch, debug and test",
+            "description": "%walkthrough.java.debugAndTest%",
             "media": {
-              "image": "walkthrough/debugger.png",
-              "altText": "Running and Debugging"
+              "image": "walkthrough/debugAndTest.png",
+              "altText": "Launch, debug and test"
             }
           },
           {
-            "id": "java.showTesting",
-            "title": "Run and debug tests",
-            "description": "The place to show all the test cases in your project. You can also run/debug your test cases from here.\n[Show Testing View](command:testExplorer.focus)",
-            "media": {
-              "image": "walkthrough/testing.png",
-              "altText": "Testing"
-            },
-            "when": "java:testRunnerActivated"
-          },
-          {
-            "id": "java.showTesting.notActivated",
-            "title": "Run and debug tests",
-            "description": "The place to show all the test cases in your project. You can also run/debug your test cases from here.",
-            "media": {
-              "image": "walkthrough/testing.png",
-              "altText": "Testing"
-            },
-            "when": "!java:testRunnerActivated"
-          },
-          {
-            "id": "java.tutorial",
-            "title": "Quick start with Java",
-            "description": "Just starting with Java? Get hands on with these tutorials to learn the fundamentals of Java editing & debugging.\n[Open Tutorials](command:toSide:java.gettingStarted)",
+            "id": "java.extensions",
+            "title": "Additional tools and frameworks",
+            "description": "%walkthrough.java.extensions%",
             "media": {
-              "image": "walkthrough/tutorials.png",
-              "altText": "Tutorials"
+              "svg": "walkthrough/extensions.svg",
```

Comment:
this SVG is pending modification, grouping extensions in categories., e.g. keymaps